src/Components/useFetch/useFetch.js

```diff
@@ -5,8 +5,8 @@ import { useState, useEffect} from 'react';
 const useFetch = () => {
 
     var apiKey = "jbfndddBALH0QmDvRJIzbDOChFEdT9PoSH08CeUV";
-    var count = 20;
-    var url =`https://api.nasa.gov/planetary/apod?api_key=${apiKey}&count=${count}`
+    var numberReq = 20;
```

Comment:
It's not an error to use var to declare and initialize variables, but I see that you are already using let and const. I would use const for these two variables numberReq and url.

Reply:
Also apiKey. Missed that in previous comment.